frameworks/cassandra/universe/config.json

```diff
@@ -515,7 +526,11 @@
             },
             "driver_name": {
               "type": "string",
-              "description": "Docker volume driver name.",
+              "description": "External Volume storage provider to use.",
+              "enum": [
+                "pxd",
+                "generic"
+              ],
```

Comment:
Since we are supporting a generic driver name, we should make it a simple string value instead of an enum. In a simple text field, the user would be able to give any driver's name.
```suggestion
```

Reply:
Good catch! thanks!